src/styles/getReportActionContextMenuStyles.js

```diff
@@ -0,0 +1,63 @@
+import CONST from '../CONST';
+import styles from './styles';
+import variables from './variables';
+import themeColors from './themes/default';
+
+/**
+ * Generate styles for the buttons in the mini comment actions menu.
+ *
+ * @param {String} [buttonState] - One of {'default', 'hovered', 'pressed'}
+ * @returns {Array}
+ */
+function getMiniButtonStyle(buttonState = CONST.BUTTON_STATES.DEFAULT) {
+    const defaultStyles = [styles.p1, styles.mv1, styles.mh2, {borderRadius: variables.componentBorderRadius}];
+    switch (buttonState) {
+        case CONST.BUTTON_STATES.HOVERED:
+            return [
+                ...defaultStyles,
+                {
+                    backgroundColor: themeColors.activeComponentBG,
+                },
+            ];
+        case CONST.BUTTON_STATES.PRESSED:
+            return [
+                styles.mv1,
+                styles.mh2,
+                {
+                    borderRadius: variables.componentBorderRadius,
+                    backgroundColor: themeColors.componentBG,
```

Comment:
All of these styles still feel strange to me after taking this for a spin. What if we just made the background color slightly darker than hover (so going from `gray1` to `gray2`) and doing nothing else, so no borderWidth or borderColor, etc.